src/components/layout/Footer.jsx

```diff
@@ -30,42 +29,52 @@ const getInvolved = [
   { name: 'Apps', path: '/get-involved/apps' }
 ];
 
-const items = partners.map((p, i) => (
-  <div key={i} className="item">
-    <Link to={`/partners/${p.name}`}>
-      <img className="-img" src={`/images/partners/${p.img}`}/>
-    </Link>
-  </div>
-));
+class Footer extends React.Component {
 
-function Footer() {
-  const menuData = [data, about, insights, getInvolved];
+  componentWillMount() {
+    this.props.getPartners();
+  }
 
-  return (
-    <footer className="c-footer">
-      <div className="footer-intro">
-        <h5 className="title"><Link to="/partners">Partners</Link></h5>
-        <div className="partners row">
-          <div className="column small-12">
-            <Carousel items={items} />
+  setPartnersList() {
+    return this.props.list.map((p, i) => (
+      <div key={i} className="item">
+        <Link to={`/about/partners/${p.id}`}>
+          <img className="-img" src={`${config.CMS_API_URL}${p.attributes.logo.thumb}`}/>
+        </Link>
+      </div>
+    ));
+  }
+
+  render() {
+    const menuData = [data, about, insights, getInvolved];
+    const items = this.setPartnersList();
+
+    return (
+      <footer className="c-footer">
+        <div className="footer-intro">
+          <h1 className="title"><Link to="/about/partners">Partners</Link></h1>
+          <div className="partners row">
+            <div className="column small-12">
+              <Carousel items={items.length ? items : [<div key="0"></div>]} />
```

Comment:
What is this => [<div key="0"></div>]? I think that it should be the Carousel itself the one in charge of doing this...